meteor-app/imports/ui/member/member-list-in.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import { List, Header } from 'semantic-ui-react'
+import MemberCardIn from './member-card-in'
+
+
+class MemberListIn extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      visibleStatus: ''
+    }
+  }
+
+toggleStatus = (id) => {
+  this.setState({
+    visibleStatus: this.state.visibleStatus == id ? '' : id
```

Comment:
With ternary operaters, it helps to put brackets around the expression to aid readability.
I must ask, however, why even bother, because all you need is for `visibleStatus` to be truthy, which you can get by assigning it to the value of id